### PR TITLE
[WIP] add measure inspector, improve element group inspector

### PR DESCRIPTION
**Pull request by jthistle**
This adds a measure inspector for when a measure is selected, and improves the element group inspector to make it collapsible and fit the inspector's look and feel better.

More features and parameters can be added to the measure inspector, this is a just a starting point.

measureinspector

As usual, any code review is very much appreciated.

### CODE REVIEW

File: mscore/inspector/inspectorGroupElement.cpp [hunk 1]
```diff
@@ -38,40 +36,16 @@ InspectorGroupElement::InspectorGroupElement(QWidget* parent)
       //
       // Select
       //
-      QLabel* l = new QLabel;
```
Comment (jthistle) [inline mscore/inspector/inspectorGroupElement.cpp]: All of this stuff was moved to the UI file.

**Comment by MarcSabatella** [comment]
Cool idea!  BTW, something I've thought about but never investigated further - any good reason offset couldn't be made part of the group inspector?  I guess maybe there is some small number of elements that don't support this (frames, ...?) but then again, not everything supports color or visibility either (again, frames, ...?)

**Comment by Jojo-Schmitz** [comment]
mtests are not happy, SIGSEGV, see https://travis-ci.org/musescore/MuseScore/jobs/541895666#L4332-L4357

**Comment by Howard-C** [comment]
This is one of the things https://musescore.org/en/node/296410 is trying to realize, but unfortunately it's a bit outdated. But we can still use the ideas and see if we can finally achieve this.

**Comment by jthistle** [comment]
Yeah, asides from being difficult to rebase, this PR in its current state is a bit broken as well...

**Comment by Howard-C** [comment]
I have been improving other inspectors for 3.4 and I hope after my end-term exams are over I can re-implement this ;-)

Let's just hope this can be in 3.5 or 3.6 (if there is one).

**Comment by dmitrio95** [comment]
The new inspector is postponed to 3.6 version so these changes could, in principle, be applied before 3.5 release. In spite of this, does it make sense to finish this PR, or should we close it and wait for an inspector redesign instead?

**Comment by elerouxx** [comment]
Hi! Any plans to reopen or implement this?
I can think on several uses for a measure inspector.
- Setting measure visibility on that staff (instead of dealing with the measure dialog)
- Overriding measure (in)visibility from Cutaway staves
- Turning on/off startBarline for measures (when previous measure is invisible its endBarline also disappears).
- Adding a clef and some kind of "staff trailer" before a visible measure on a cutaway score (cutaway scores need to show clefs for the first time and in other situations too).
...

**Pull request closed without merging**